src/PostalRegistry.Api.Legacy/PostalInformation/Responses/PostalInformationLinkedDataEventStreamResponse.cs

```diff
@@ -74,25 +146,24 @@ public class PostalInformationVersionObject
 
         [DataMember(Name = "postnaam", Order = 7)]
         [JsonProperty(Required = Required.AllowNull, NullValueHandling = NullValueHandling.Ignore)]
-        public List<LanguageValue>? PostalNames { get; set; }
```

Comment:
In a response, when empty isn't it better to have an empty list instead of null?
If yes, will have to change `[JsonProperty(Required = Required.AllowNull` to `Required`

Reply:
in JSON-LD it is common practice that when a value is empty or null it is not published